02-basic-typescript/bubble/src/components/Bar.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './Bar.css';
+
+
+type BarProps = {
+  value: number,
+  maxValue: number,
+};
+
+
+export const Bar = ({
+  value,
+  maxValue,
+}: BarProps): React.ReactElement => {
+  return(<div
+    className="bar"
+    style={{
+      height: `${value / maxValue * 100}%`
+    }}
+  >
+    {/* {value} */}

```

Comment:
Is it needed?

Reply:
I tried to display bar value. But it doesn't look nice with large arrays. I'll clean up it.